app/main.ipynb

```diff
@@ -25,18 +25,20 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "import sys\n",
+    "import random\n",
```

Comment:
The `random` library is imported but not used in the code. Consider removing this import to clean up the code.